### DISCUSSION TITLE
萌黄公告

### DISCUSSION THREAD

**Discussion by Moment1745** [share121/inter-knot#2188] [其他]
![Image_1723699552367](https://github.com/user-attachments/assets/a71a19ee-6ac8-48b8-aeff-cd11cec1fc5f)
众望所归的解斑削弱来了!这个削弱能让解斑到t几呢？详情请关注火影忍者手游情报社。火之意志，生生不息

**Comment by JiangDonghuazhu** [comment]
t10086

**Comment by Will-Dyt** [comment]
TD

**Comment by anything12121** [comment]
火影忍者就这么入侵了？？？

**Reply by Usodaru48763** [reply]
哪都有村裡人
